integration/clean/clean_aws_resource.go

```diff
@@ -0,0 +1,154 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build clean
+// +build clean
+
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	ec2Type "github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	ssmType "github.com/aws/aws-sdk-go-v2/service/ssm/types"
+	smithyTime "github.com/aws/smithy-go/time"
+)
+
+const (
+	daysToKeep   = 30
+	keepDuration = -1 * time.Hour * 24 * time.Duration(daysToKeep)
+)
+
+func main() {
+	var expirationDate = time.Now().UTC().Add(keepDuration)
+
+	log.Println("Begin to clean AWS resources.")
+
+	cleanAMI(expirationDate)
+	cleanSSMParameterStore(expirationDate)
+
+	log.Println("Finished cleaning AWS resources.")
+}
+
+func cleanSSMParameterStore(expirationDate time.Time) {
+	log.Println("Begin to clean SSM Parameter Store")
+
+	ctx := context.Background()
+	defaultConfig, err := config.LoadDefaultConfig(ctx)
+
+	if err != nil {
+		log.Fatalf("Load default config failed because of %v", err)
+	}
+
+	ssmClient := ssm.NewFromConfig(defaultConfig)
+
+	//Allow to load all th since the default respond is paginated auto scaling groups.
```

Comment:
Typo?

Reply:
Yes. My fault for this typo. Fixed it with the newest commit